src/actions/blockActions.ts

```diff
@@ -164,8 +164,13 @@ export function fetchBlocks(page = 1, chain?: string) {
           }
         }),
       )
-
-      const cleanedBlocks = res.flat().filter(r => r !== undefined) as Block[]
+      let cleanedResults = res.filter(r => r['status'] === 'fulfilled')
+      // @ts-ignore
+      cleanedResults = cleanedResults.map(r => r['value'])
+      // @ts-ignore
+      const cleanedBlocks = cleanedResults
+        .flat()
+        .filter(r => r !== undefined) as Block[]
```

Comment:
to avoid @ts-ignore
```suggestion
const cleanedBlocks = res
        .filter(it => it.status === 'fulfilled' && it.value)
        .map(it => (it as PromiseFulfilledResult<Block[]>).value)
        .flat() as Block[]
```

Reply:
Thanks @aldo-maciel :heart: